File: program_data/github_java_sort_function/4/41.java
 public void sort(int eCost[])
    {
        int n = eCost.length;
 
        for (int i = n / 2 - 1; i >= 0; i--)
            heapify_eC(eCost, n, i);
 
        
        for (int i=n-1; i>=0; i--)
        {
            
            int temp = eCost[0];
            eCost[0] = eCost[i];
            eCost[i] = temp;
            heapify_eC(eCost, i, 0);
        }
    }
 
    void heapify_eC(int sorting[], int n, int i)
    {
        int root = i;  
        int r = 2*i + 2;  
        int l = 2*i + 1; 
         
        if (l < n && sorting[l] > sorting[root])
            root = l;
 
        if (r < n && sorting[r] > sorting[root])
            root = r;
         
        if (root != i)
        {
            int temp = sorting[i];
            sorting[i] = sorting[root];
            sorting[root] = temp;
            heapify_eC(sorting, n, root);
        }
    }

File: program_data/PKU_raw/94/41.c
int main()
{
	int a [ 500 ], b [ 500 ];
	int i, j = 0, k, l, temp, n, p;
	cin >> n;
	for ( i = 0; i < n; i ++ )
	{
		cin >> l;
		a [ i ] = l;
		if ( a [ i ] % 2 != 0 )
		{
			b [ j ] = a [ i ];
			j ++;
		}
	}
	for ( p = 0; p < j - 1; p ++ )
	for ( k = 0; k < j - 1 - p; k ++ )
	{
		if ( b [ k ] > b [ k + 1 ] )
		{
			temp = b [ k ]; b [ k ] = b [ k + 1 ]; b [ k + 1 ] = temp; 
		}
	}
	cout << b[0];
	for ( p = 1; p < j; p ++ )
	{
		cout << "," << b[p];
	}
	return 0;
}